horreum-backend/src/main/java/io/hyperfoil/tools/horreum/svc/LabelValuesService.java

```diff
@@ -531,6 +522,13 @@ protected static List<ExportedLabelValues> parse(List<Object[]> nodes) {
                                     return exportedLabelValue;
                                 })));
 
+        if (limit != null && limit > 0) {
+            page = (page == null || page < 0) ? 0 : page;
+            List<List<ExportedLabelValues>> partitioned = Lists.partition(exportedLabelValues.values().stream().toList(),
```

Comment:
Can we use Collectors.partitioning/groupingBy instead of Guava Lists?

Reply:
Good suggestion, let me see if that works since it will avoid to add the new guava dependency only for this

Reply:
@stalep looks like I was able to get rid of `guava` as I don't actually need to partition the records.. I just need to skip the first `limit * page` elements and keep the other `limit` ones:
```java
        if (limit != null && limit > 0) {
            long toSkip = (long) limit * ((page == null || page < 0) ? 0 : page);
            return exportedLabelValues.values().stream().skip(toSkip).limit(limit).collect(Collectors.toList());
        }
```

wdyt?